azurelinuxagent/common/logcollector.py

```diff
@@ -339,26 +335,35 @@ def collect_logs(self):
         try:
             # Clear previous run's output and create base directories if they don't exist already.
             self._create_base_dirs()
-            LogCollector._reset_file(_OUTPUT_RESULTS_FILE_PATH)
+            LogCollector._reset_file(OUTPUT_RESULTS_FILE_PATH)
+            start_time = datetime.utcnow()
+            _LOGGER.info("Starting log collection at {0}".format(start_time.strftime("%Y-%m-%dT%H:%M:%SZ")))
 
             files_to_collect = self._create_list_of_files_to_collect()
             _LOGGER.info("### Creating compressed archive ###")
 
-            with zipfile.ZipFile(_COMPRESSED_ARCHIVE_PATH, "w", compression=zipfile.ZIP_DEFLATED) as compressed_archive:
-                for file in files_to_collect:
-                    archive_file_name = LogCollector._convert_file_name_to_archive_name(file)
-                    compressed_archive.write(file, arcname=archive_file_name)
+            with zipfile.ZipFile(COMPRESSED_ARCHIVE_PATH, "w", compression=zipfile.ZIP_DEFLATED) as compressed_archive:
```

Comment:
Thanks for the analysis! Since we are first targeting VMs with systemd (in order to enable resource limiting), none of them actually run py2.6, so I will take this change when we are expanding to non-systemd VMs. This will be useful for sure!